pontoon/base/models.py

```diff
@@ -76,6 +76,10 @@ def cldr_plurals_list(self):
         else:
             return map(int, self.cldr_plurals.split(','))
 
+    @property
+    def dir(self):
```

Comment:
That's also true. And - we might not even need the dir property, because we can just use dir=auto?
https://bugzilla.mozilla.org/show_bug.cgi?id=1190566#c4